extensions/gitpod/package.json

```diff
@@ -99,6 +99,7 @@
 		"@types/yazl": "^2.4.2"
 	},
 	"dependencies": {
+		"gitpod-shared": "0.0.1",
```

Comment:
We should not change gitpod desktop extension at all! Only gitpod-web and gitpod-remote via gitpod-shared.

Reply:
updated

Reply:
I try to open workspace in code Desktop to test track event, found it not works, so add a support for `gitpod-desktop` 😅 https://github.com/gitpod-io/openvscode-server/compare/b78c0eebd26272f7df9c76fe03a0eea15658342d..4fc2b5675dcee25b9375585fd8fd18887a186134

I'll try again, teach me if I miss some steps

Reply:
Ahh, I should re-install gitpod-remote in Desktop. Correct?

Reply:
If it's correct, we should write this down somewhere👍 cc @akosyakov

Reply:
We need to run `npx gulp package-gitpod-marketplace-extensions` to package `remote`. This one need doc too